cirq-core/cirq/ops/raw_types.py

```diff
@@ -36,8 +36,20 @@
 import numpy as np
 
 from cirq import protocols, value
+from cirq._import import LazyLoader
 from cirq.type_workarounds import NotImplementedType
 
+# Lazy imports to break circular dependencies.
+controlled_gate = LazyLoader("controlled_gate", globals(), "cirq.ops.controlled_gate")
+controlled_operation = LazyLoader(
+    "controlled_operation", globals(), "cirq.ops.controlled_operation"
+)
+gate_operation = LazyLoader("gate_operation", globals(), "cirq.ops.gate_operation")
+linear_combinations = LazyLoader("linear_combinations", globals(), "cirq.ops.linear_combinations")
+line_qubit = LazyLoader("line_qubit", globals(), "cirq.devices.line_qubit")
+random_gate_channel = LazyLoader("random_gate_channel", globals(), "cirq.ops.random_gate_channel")
```

Comment:
Optional: Since five out of the six modules imported here are in the cirq.ops package, you could reduce the number of imports by importing cirq.ops. Up to you.

Reply:
Also might want to write down what the circular path is ?

Reply:
@MichaelBroughton  you mean like describe why it comes back here?  A lot of these are just coming from places where we've let the basic data structures, Gate, Qid, grow hair (like why do I need a line qubit in the definition of gate? because we use it in pow which then relies on the decompose protocol which needs to reference qubits)

Reply:
Updated to only have ops.

Reply:
I was thinking something along the lines of a comment at the top:
`# moments -> circuits -> moments`
`circuits = LazyLoader("circuits", globals(), "cirq.circuits.circuit")`

So people can see the path, but I guess it's not super important.

Reply:
Tried it.  These all tend to be just simple circles, so I don't think they add much.  Going to skip adding.

Reply:
SG